app/views/form-designer/pages/preview-question.html

```diff
@@ -1,49 +1,65 @@
+    {# IF ‘TITLE’ IS SELECTED BUT NOT ‘FIRST, MIDDLE AND LAST NAME’ AND Q IS NOT OPTIONAL THEN: #}
+    {% set answerTypeInformation = 'People will be able to use any characters in their answers - including special characters. They can leave ‘title’ empty if they need to.' %}
+  {% elif pageData['input'] === 'single-field' %}
+    {# IF ‘FULL NAME IN A SINGLE BOX’ IS SELECTED BUT NOT ‘TITLE’ THEN: #}
+    {% set answerTypeInformation = 'People will be able to use any characters in their answer - including special characters.' %}
+  {% else %}
+    {# OTHERWISE: #}
+    {% set answerTypeInformation = 'People will be able to use any characters in their answers - including special characters.' %}
   {% endif %}
 {% elif pageData['type'] === 'companyName' %}
-  {% set answerType = 'Company or organisation’s name' %}
+  {% set answerTypeInformation = 'People will be able to use any characters in their answer - including special characters.' %}
 {% elif pageData['type'] === 'email' %}
-  {% set answerType = 'Email address' %}
+  {% set answerTypeInformation = 'People will only be able to enter an answer that’s in the format of an email address, like name@example.com.' %}
 {% elif pageData['type'] === 'phone' %}
-  {% set answerType = 'Phone number' %}
+  {% set answerTypeInformation = 'People will be able to enter between 8 and 15 characters. They can use numbers, spaces, ‘ext’, ‘-’, ‘+’, ‘(’ and ‘)’.' %}
 {% elif pageData['type'] === 'national-insurance-number' %}
-  {% set answerType = 'National Insurance number' %}
+  {% set answerTypeInformation = 'People will only be able to enter an answer that’s in the format of a National Insurance number - 2 letters, 6 numbers, then A, B, C or D.' %}
 {% elif pageData['type'] === 'address' %}
-  {% set inputTypeTitle = data.addressInputTypeTitle %}
-  {% if ('uk-address' in pageData['input']) and ('international-address' in pageData['input']) %}
-    {% set inputType = 'UK and international addresses' %}
-  {% elif 'uk-address' in pageData['input'] %}
-    {% set inputType = 'UK addresses' %}
-  {% elif 'international-address' in pageData['input'] %}
-    {% set inputType = 'International addresses' %}
+  {% if ('uk-address' in pageData['input']) and not ('international-address' in pageData['input']) %}
+    {# IF UK SELECTED: #}
+    {% set answerTypeInformation = 'People will only be able to enter a postcode that’s in the format of a UK postcode.' %}
+  {% else %}
+    {# IF INTERNATIONAL SELECTED: #}
+    {% set answerTypeInformation = 'People will be able to answer using the appropriate format for the address. This is useful if you expect any international addresses. ' %}
   {% endif %}
 {% elif pageData['type'] === 'date' %}
-  {% set inputTypeTitle = data.dateOfBirthInputTypeTitle %}
   {% if pageData['input'] === 'yes' %}
-    {% set inputType = 'Yes' %}
+    {# IF DATE OF BIRTH: #}
+    {% set answerTypeInformation = 'People will only be able to enter a date that is in the past.' %}
```

Comment:
```suggestion
    {% set answerTypeInformation = 'People will only be able to enter a date that’s in the past.' %}
```
My bad: "that is" should be contracted to "that’s".